vercel.json

```diff
@@ -1,9 +1,4 @@
 {
-  "functions": {
```

Comment:
What was the reason for removing this? I'm asking as I'm not at all familiar with the code, but I did gather that the people hr feed gets cached when debugging why the jobs weren't displaying.

Reply:
This method of caching by specifying a file path to read / write data from is no longer supported by Vercel - I believe this was ready to be removed when Ben removed our PeopleHR cache system a while ago.